pkg/server/registry/apigroups/acorn/apps/validator.go

```diff
@@ -96,6 +96,12 @@ func (s *Validator) Validate(ctx context.Context, obj runtime.Object) (result fi
 		return
 	}
 
+	au := app.Spec.AutoUpgrade
+	if au != nil && !*au && autoupgrade.ImpliedAutoUpgrade(app.Spec.AutoUpgradeInterval, app.Spec.Image) {
```

Comment:
Should this be:
```suggestion
	if (au == nil || !*au) && autoupgrade.ImpliedAutoUpgrade(app.Spec.AutoUpgradeInterval, app.Spec.Image) {
```

Reply:
Yes! It should be.

Reply:
Added this and updated a test that failed as a result of this being added.